config.go

```diff
@@ -35,6 +35,7 @@ const (
 	defaultImagePath            = "/usr/share/clear-containers/clear-containers.img"
 	defaultHypervisorPath       = "/usr/bin/qemu-lite-system-x86_64"
 	defaultProxyURL             = "unix:///run/cc-oci-runtime/proxy.sock"
+	defaultPauseBinPath         = "/tmp/bundles/pause_bundle/rootfs/bin/pause"
```

Comment:
@sboeuf it would be better to add the pause bundle path as an optional runtime config and have a sane default for it (`/var/lib/clearcontainers/runtime/bundles/pause_bundle` looks ok).

Then the pause binary path should be relative to that and although I agree with @jodh-intel about it being quite generic, the pause bundle is explicitly made of one single binary file so the risks of seeing a conflict with the rest of the bundle content is really low. Given that `bin/pause` is a k8s accepted standard, I think we're pretty safe keeping it as is.